components/Press/PressLinks/__tests__/PressLinks.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import { mount, shallow } from 'enzyme';
+
+import PressLinks from '../PressLinks';
+import LinkGroup from '../LinkGroup';
+
+describe('PressLinks', () => {
+  test('it should render properly no props', () => createShallowSnapshotTest(<PressLinks />));
+});
+
+describe('PressLinks > LinkGroups', () => {
+  test('it should render properly with required props', () =>
```

Comment:
😓 this is my fault for not having a standard throughout the app...

I'm eventually going to make every `test()` into `it()`

For now, can you drop the "it" in the test name?